en/integrations/plugin-packs/procedures/network-switchs-symbol-wing-snmp.md

```diff
@@ -0,0 +1,179 @@
+| device.memory.usage.percentage |             | %    |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prerequisites
+
+No specific prerequisites
+
+### Network flow
+
+The target server must be reachable from the Centreon Poller on the UDP/161 SNMP
+port.
+
+## Setup
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Online License-->
+
+1. Install the Centreon Plugin package on every Centreon poller expected to monitor *Symbol WiNG* ressources:
+
+```bash
+yum install centreon-plugin-Network-Symbol-Wing-Snmp
+```
+
+2. On the Centreon Web interface, install the *Symbol WiNG switch SNMP* Centreon Plugin Pack on the **Configuration > Plugin Packs** page
+
+<!--Offline IMP License-->
+
+1. Install the Centreon Plugin package on every Centreon poller expected to monitor *Symbol WiNG* ressources:
+
+```bash
+yum install centreon-plugin-Network-Symbol-Wing-Snmp
+```
+
+2. Install the *Symbol WiNG switch SNMP* Centreon Plugin Pack RPM on the Centreon Central server:
+
+ ```bash
+yum install centreon-pack-network-switchs-symbol-wing-snmp
+```
+
+3. On the Centreon Web interface, install the *Symbol WiNG switch SNMP* Centreon Plugin Pack on the **Configuration > Plugin Packs** page
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Configuration
+
+### Host
+
+* Log into Centreon and add a new Host through **Configuration > Hosts**.
+* Fill the "Name", "Alias" & "IP Address / DNS" fields according to your *Symbol WiNG* server settings
+* Select the **network-switchs-symbol-wing-snmp-custom** template to apply to the Host.
+
+If you are using SNMP Version 3, use the SNMPEXTRAOPTIONS Macro to configure
+    your own SNMPv3 credentials combo.
+
+| Mandatory | Name             | Description                                              |
+|:----------|:-----------------|:---------------------------------------------------------|
+|           | SNMPEXTRAOPTIONS | (Default: 'Configure your own SNMPv3 credentials combo') |
+
+## How to check in the CLI that the configuration is OK and what are the main options for ? 
+
+Once the plugin installed, log into your Centreon Poller CLI using the 
+*centreon-engine* user account and test the Plugin by running the following 
+command:
+
+```bash
+/usr/lib/centreon/plugins//centreon_symbol_wing.pl \
+    --plugin=network::symbol::wing::snmp::plugin \
+    --mode=systems \
+    --hostname='10.0.0.1' \
+    --snmp-community='my-snmp-community' \
+    --snmp-version='2c' \
+    --filter-name='' \
+    --warning-cpu-utilization-1m='' \
+    --critical-cpu-utilization-1m='' \
+    --warning-cpu-utilization-5m='80' \
+    --critical-cpu-utilization-5m='90' \
+    --warning-cpu-utilization-15m='' \
+    --critical-cpu-utilization-15m='' \
+    --warning-memory-usage='' \
+    --critical-memory-usage='' \
+    --warning-memory-usage-free='' \
+    --critical-memory-usage-free='' \
+    --warning-memory-usage-prct='' \
+    --critical-memory-usage-prct='' \
+    --warning-devices-total='' \
+    --critical-devices-total='' \
+    --use-new-perfdata 
```

Comment:
--use-new-perfdata is useless in the _systems_ mode as the new format is forced (see here https://github.com/centreon/centreon-plugins/blob/a8b6ea80d78b49a13aa9245a5397f9179ef8a20f/network/symbol/wing/snmp/mode/systems.pm#L141)
```suggestion

```